domains/Accounts/Tests/Feature/EmailVerificationTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Domains\Accounts\Tests\Feature;
+
+use Carbon\Carbon;
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Faker\Factory;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class EmailVerificationTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected User $user;
+    protected Generator $faker;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user = UserFactory::new()->unverified()->create();
+        $this->faker = Factory::create();
+    }
+
+    /** @test */
+    public function it_fails_to_validate_a_users_email_on_link_hash_mismatch(): void
+    {
+        $this->get(route('accounts.verify',
+            $this->user->id,
+            base64_encode(Hash::make($this->faker->safeEmail))
+        ))->assertResponseStatus(Response::HTTP_FORBIDDEN);
+
+        $this->seeInDatabase($this->user->getTable(), [
```

Comment:
Let's use the `users` table name, directly, instead of relying on the `getTable()` method.

Reply:
Ok, it was just that if for some reason (although, I admit, very very unlikely) we change the table name, this test will need updating.

Reply:
And that's ok. We want the tests to break if we change the table name. 

In a new greenfield project, this might seem unnecessary, but in the context of continuous integration, and the app being in production, we can't simply change the table names :smile:

Reply:
Can we switch from calling `getTable()` to use the table name, directly?